bcda/auth/alpha.go

```diff
@@ -8,35 +8,47 @@ import (
 
 	"github.com/CMSgov/bcda-app/bcda/database"
 	"github.com/CMSgov/bcda-app/bcda/models"
-	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go"
 	"github.com/pborman/uuid"
 	log "github.com/sirupsen/logrus"
 )
 
 type AlphaAuthPlugin struct{}
 
 func (p AlphaAuthPlugin) RegisterClient(localID string) (Credentials, error) {
-
+	regEvent := event{op: "RegisterClient", trackingID: localID}
```

Comment:
I wish the struct was easier to work with.  There are some hints from the IDE about what keys might be possible, and when you command+hover over `event` it shows you a definition, but I'd like it even better if this was chain-able like:
`regEvent := event.New().op("RegisterClient").trackingID(localID)`
or functional like:
`regEvent := event.New(event.Op("RegisterClient"), event.TrackingID(localID))`

Maybe in a far-off refactor.